docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1714 @@
+{
```

Comment:
Yes, but I think it cannot raise the error when user set `include=np.full((k, something), 1, dtype=bool)` ? because this one has length `k` as well. Are we okay with this? Or, should I check the `.ndim` seperately?